web/styles/popovers.css

```diff
@@ -1067,8 +1067,10 @@ ul {
     }
 }
 
-.visibility_policy_popover {
-    padding: 0 4px;
+.visibility-policy-popover {
+    ul.popover-menu-list {
```

Comment:
It is necessary to include the `ul` in this selector? Just the class should suffice, I'd think.

Reply:
Makes sense, removed the `ul` portion of the selector. Thanks!